src/types/submit.ts

```diff
@@ -0,0 +1,21 @@
+import { z } from 'zod'
+
+const nonEmptyStringContraint = z.string().min(1)
+const EmptyStringContraint = z.string()
+
+export const SubmitSchema = z.object({
+  firstName: nonEmptyStringContraint,
+  email: nonEmptyStringContraint,
```

Comment:
Zod string objects has a built-in method for checking the email syntax:
```suggestion
  email: nonEmptyStringContraint.email(),
```